lib/radar_client_rb/resource.rb

```diff
@@ -22,13 +22,29 @@ def get
       @client.redis.hgetall(@name).each do |key, value|
         user_id, client_id = key.split('.')
         message = JSON.parse(value)
-        if message['online']
+        if message['online'] && is_sentry_online?(message['sentry'])
           result[user_id] ||= { :clients => {}, :userType => message['userType'] }
           result[user_id][:clients][client_id] = message['userData'] || {}
         end
       end
       result
     end
+
+    private
+
+    def sentries
+      @sentries ||= @client.redis.hgetall('sentry:/radar')
```

Comment:
Instead of hgetall, maybe we should extract all the sentries out and do an hmget on this key. We are probably only interested in very few servers.